bidwire/bid.py

```diff
@@ -17,36 +17,36 @@ class Bid(Base):
     location = Column(Text)
     open_date = Column(DateTime)
     items = Column(JSON)
+    site = Column(Text)
```

Comment:
Let's add in this file a new `Enum` for our 'site' values. That way we have a consistent way to get the string to put in the database, and in all other places we need to pass around a 'site', we can pass the enum.

It would look something like this:
```python
from enum import Enum
class Bid:
  class Site(Enum):
      COMMBUYS = 1
      CITYOFBOSTON = 2

  # then, where 'site' is used...
  def get_new_identifiers(identifiers, site):
     # site is of type Site, not string
     [...]
     query = .... Bid.site == site.name ...
```

The we can also do, outside of the Bid class, e.g. in the Alembic migration...
```python
[...] server_default=Bid.Site.COMMBUYS.name
```

(More about Python Enums here: https://docs.python.org/3/library/enum.html)